vergedb_test/routes/drones.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+const router = express.Router();
+const drones = require('../services/drones');
+
+/* GET drones listing. */
+router.get('/', function (req, res, next) {
+    try {
+        //Commands here
+        res.json(drones.getMultiple(req.query.page));
+        
+    } catch (err) {
```

Comment:
- [ ] Instead of printing the error to a console, create a response error.
- [ ] On success, we need to return the appropriate http response code